### ISSUE TITLE
LYWSD03MMC is not working after update to 7.5.0

### ISSUE THREAD

**Issue by MiguelCosta** [custom-components/ble_monitor#697]
Hi

after update the ble_monitor from 7.4.0 to 7.5.0, the LYWSD03MMC devices are not showing the values in the HA.

### Before update

![image](https://user-images.githubusercontent.com/645482/152363073-4eb101e9-a73d-4308-80ee-4d24bfdbf66e.png)

### After update

![image](https://user-images.githubusercontent.com/645482/152363409-795dd7e9-1921-4fe4-8fd1-3d95574c3706.png)

## Notes

- Home Assistant Core version: core-2021.12.10
- After rollback to 7.4.0 version everything goes back to normal
- I don't have any log error

anyone aware?

**Comment by Ernst79** [comment]
Do you have more sensors than this, that do work?

**Comment by MiguelCosta** [comment]
hi @Ernst79 

I only have LYWSD03MMC sensors

![image](https://user-images.githubusercontent.com/645482/152367163-edcdf5ab-22d3-4d74-8781-e36629aab8bd.png)

(print using 7.4.0 version)

**Comment by jovanotti25** [comment]
I have same problem with uptdate   7.5.0 e i have more sensors and don´t work

**Comment by Magalex2x14** [comment]
Data from my LYWSD03MMC comes after the update as before (on two systems).
Guys, how long did it take after the update before you started to worry?
@jovanotti25 what firmware do you have on your sensors? Official from Xiaomi?

**Comment by Ernst79** [comment]
@jovanotti25, Do you also only have LYWSD03MMC sensors?

**Comment by jovanotti25** [comment]
No i have more 
![sensor](https://user-images.githubusercontent.com/67195538/152368945-080c8d07-a68d-4407-a45b-d44bd2cbede2.png)

**Comment by jovanotti25** [comment]
and this 
![image](https://user-images.githubusercontent.com/67195538/152369224-3c823f08-2fbc-4814-b6f5-1616abf8f930.png)

**Comment by Ernst79** [comment]
Could one of you enable debug logging, to see if it shows some hints what is wrong. I don't have issues myself. 

And please try to open the ble_monitor options and close it afterwards.

And how did you configure BLE monitor, in YAML or in the UI?

**Comment by skynetua** [comment]
I have configured via UI and had this issue and constant exception in logs:
```
2022-02-03 17:29:35 ERROR (Thread-181) [root] Uncaught thread exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/config/custom_components/ble_monitor/__init__.py", line 704, in run
    if (self.config[CONF_BT_AUTO_RESTART] is True) and (interface_is_ok[hci] is False):
KeyError: 'bt_auto_restart'
```
After options re-submit the problem has gone.

**Comment by jovanotti25** [comment]
i have configured via UI 

![image](https://user-images.githubusercontent.com/67195538/152376819-6cce16b9-0a9c-4131-aac8-9e930a9bd425.png)

**Comment by Ernst79** [comment]
This is the same issue as #695. Can be solved by opening the ble_monitor options and click submit. I'll create a fix as well that updates the config automatically upon restart, after the update.

**Comment by Magalex2x14** [comment]
Did you, guys, update HA to 2022.2.0 before the ble_monitor update?

**Comment by Ernst79** [comment]
Could one of you try to add the changes in #698 manually. 

After that, restart HA and check that the issue is solved. 

It will add the missing `bt_auto_restart` to the config.

**Comment by jovanotti25** [comment]
I update now and make issue #695 and works thanks for your support keep good work with this integration

**Comment by MiguelCosta** [comment]
After options re-submit the problem has gone.

![image](https://user-images.githubusercontent.com/645482/152381307-80a0dd4e-15f2-4f4b-8811-6e41573eee65.png)

**Comment by Ernst79** [comment]
Fixed in 7.5.1

**Comment by josecarre** [comment]
> 

Yep, I confirm is working again with 7.5.1, thanks!
